.vscode/settings.json

```diff
@@ -2,5 +2,8 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": "explicit"
   },
+  "[yaml][markdown][html][scss]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
i assume you'll later replace this with biome in #27

Reply:
yes

Reply:
biome cannot format these files, so i'm setting these to make it cooperate with prettier when i do add it